app/views/posts/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>show the posts of a user</h1>
\ No newline at end of file
+<div class="user">
+  <div class="user_data">
+    <h3><%= @user.name%></h3>
+    <p>Number of posts: <%= @user.posts_counter %></p>
+  </div>
+</div>
+
+<% @posts.each do |post| %>
+  <%= render post %>
+<%end%>
+
```

Comment:
Please use `recent_5_comments` from Post model to render 5 most recent comments in the `app/views/posts/index.html.erb` page.